src/stylesheets/common/_helpers.scss

```diff
@@ -172,14 +172,13 @@
   color: #333333;
 }
 
-.text-16 {
-  // font-weight: normal;
-  // font-size: 16px;
-  // line-height: 1.6;
+.deemphasize-text {
```

Comment:
And yeah, a section in the styleguide on different text treatments (and maybe background colors, too) would be a good addition.